core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/dot-ema-shell.component.ts

```diff
@@ -230,25 +137,22 @@ export class DotEmaShellComponent implements OnInit, OnDestroy {
         this.#siteService.switchSite$.pipe(skip(1)).subscribe(() => {
             this.#router.navigate(['/pages']);
         });
-
-        // We need to check if the language is translated
-        this.store.translateProps$
-            .pipe(takeUntil(this.#destroy$))
-            .subscribe(({ languages, page, pageLanguageId }) => {
-                const currentLanguage = languages.find((lang) => lang.id === pageLanguageId);
-
-                if (!currentLanguage.translated) {
-                    this.createNewTranslation(currentLanguage, page);
-                }
-            });
     }
 
     ngOnDestroy(): void {
         this.#destroy$.next(true);
         this.#destroy$.complete();
+        this.translatePageEffect.destroy();
```

Comment:
@zJaaal Calling destroy for `translatePageEffect` is not necessary if the effect is created in the constructor or like a class attr.